sentry-android-core/src/main/java/io/sentry/android/core/AndroidContinuousProfiler.java

```diff
@@ -143,17 +159,35 @@ private synchronized void stop(final boolean restartProfiler) {
 
     // check if profiler end successfully
     if (endData == null) {
+      // A problem occurred. Profile chunk is not captured. Let's reset ids.
+      chunkId = SentryId.EMPTY_ID;
+      profilerId = SentryId.EMPTY_ID;
```

Comment:
```suggestion
      if (!restartProfiler) {
        profilerId = SentryId.EMPTY_ID;
      }
```
should it be like this?

Reply:
not sure
in this case we don't restart profile chunks, so the profiler effectively stops
we could also restart the profiler in here, and we could keep the same profiler id, but that would create a 10 second gap in the profile... 
not sure honestly, but we are talking about an edge case that should never happen (profiler didn't write a file or it never started in the first place)

Reply:
ok, i will just continue the execution and log an error message without returning early
i realized the `isRunning` flag is also not reset, and unexpected behaviours may happen: if a user starts a profile he expects to have it run continuously until he stops it, even in case some error occurs, imho

Reply:
fine by me!